### PR TITLE
Bug 1866331: Add field level help at various OCS pages

### PR DESCRIPTION
**Pull request by bipuladh**
- Adds help text in dashboards.
 - Adds help text in installation flow.
Screenshot from 2021-01-12 18-55-24
Screenshot from 2021-01-12 18-14-08
Screenshot from 2021-01-12 18-13-57
Screenshot from 2021-01-12 18-09-13

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx [hunk 1]
```diff
@@ -79,7 +80,14 @@ const DataConsumptionCard: React.FC = () => {
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>{t('noobaa-storage-plugin~Performance')}</DashboardCardTitle>
+        <DashboardCardTitle>
+          {t('noobaa-storage-plugin~Performance')}
+          <FieldLevelHelp>
+            {t(
+              'noobaa-storage-plugin~Shows an overview of the data consumption per provider or account collected from the day of the entity creation.',
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx:87]:
```suggestion
              'noobaa-storage-plugin~Shows an overview of the data consumption per provider or per account from the day of the entity creation.',
```

**Comment by cloudbehl** [comment]
frontend tests are failing.

File: frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.tsx [hunk 2]
```diff
@@ -191,7 +195,14 @@ const BreakdownCard: React.FC = () => {
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>{t('noobaa-storage-plugin~Capacity breakdown')}</DashboardCardTitle>
+        <DashboardCardTitle>
+          {t('noobaa-storage-plugin~Capacity breakdown')}
+          <FieldLevelHelp>
+            {t(
+              'noobaa-storage-plugin~This card shows used capacity by different resources. The available capacity is based on cloud services therefore it cannot be shown.',
+            )}
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.tsx:203]: Lets be consistent, We are not mentioning card anywhere.
```suggestion
              'noobaa-storage-plugin~Shows used capacity by different resources. The available capacity is based on cloud services therefore it cannot be shown.',
            )}
```

**Review by afreen23 — CHANGES_REQUESTED** [review]
Please rebase

**Pull request closed without merging**